WhatsNew.Cli/README.md

```diff
@@ -29,51 +29,51 @@ Metrics for the "what's new" pages are made available at [aka.ms/whatsnewindocs]
 1. Store the PAT in an environment variable named `GitHubKey`.
 1. Set up an OSPO personal access token
    > This step is required for versions 2.0 and above.
 
    The OSPO personal access token is required to access the OSPO API. This API determines community contributors and Microsoft employee and vendor contributors.
 1. Request a token at [Visual Studio Online](https://ossmsft.visualstudio.com/_usersSettings/tokens). You can disable all scopes *except* read:user profile.
-1. Store the token in an environment variable named "OSPO_KEY".  If you are using the GitHub Action to generate the PR automatically, add the key as a secret in your repository:
+1. Store the token in an environment variable named "OSPO_KEY". **If you are using the GitHub Action to generate the PR automatically**, add the key as a secret in your repository:
    - Go to **Settings** on your repo.
    - Select **Secrets**
    - Add "OSPO_KEY" as a **Repository Secret**.
 1. Install the [.NET SDK 7.0](https://dotnet.microsoft.com/download/dotnet) or later.
-1. build and publish the tool:
+1. build and publish the tool. The `WhatsNew.Cli` folder is at the root of the repository you closed in step 1.
 
     ```bash
-    cd WhatsNew.Cli
+    cd ./WhatsNew.Cli
     dotnet build
     dotnet publish
     ```
 
-1. Run the appropriate `dotnet whatsnew` command. See the examples below. The location of the generated Markdown file can be specified by setting the `--savedir` option:
+1. Run the appropriate `dotnet whatsnew` command. See the examples below:
 
    ```bash
    cd work/visualstudio-docs-pr
-   dotnet ../docs-tools/WhatsNew.Cli/bin/Debug/net7.0/publish/WhatsNew.Cli.dll --owner MicrosoftDocs --repo visualstudio-docs-pr --savefile ./docs/ide/whats-new-visual-studio-docs.md 
+   dotnet ../docs-tools/WhatsNew.Cli/bin/Debug/net7.0/publish/WhatsNew.dll --owner MicrosoftDocs --repo visualstudio-docs-pr --savefile ./docs/ide/whats-new-visual-studio-docs.md 
    ```
 
 ### Examples
 
 **Display the help menu:**
 
 ```bash
-dotnet whatsnew -h
+dotnet WhatsNew.dll -h
 ```
 
 **Generate the *dotnet/AspNetCore.Docs* repo's what's new page for the period starting 5/1/2020 and ending 5/31/2020. Process PRs in the *dev* branch.**
 
 ```bash
-dotnet WhatsNew.Cli.dll --owner dotnet --repo AspNetCore.Docs --branch dev --startdate 2020-05-01 --enddate 2020-05-31
+dotnet WhatsNew.dll --owner dotnet --repo AspNetCore.Docs --branch dev --startdate 2020-05-01 --enddate 2020-05-31
 ```
 
 **Generate the Cognitive Services docset's what's new page for the period starting 5/1/2020 and ending 5/31/2020. Process PRs in the repository's default branch.**
 
 ```bash
-dotnet WhatsNew.Cli.dll --owner MicrosoftDocs --repo azure-docs-pr --docset cognitive-services --startdate 2020-05-01 --enddate 2020-05-31
+dotnet WhatsNew.dll --owner MicrosoftDocs --repo azure-docs-pr --docset cognitive-services --startdate 2020-05-01 --enddate 2020-05-31
```

Comment:
Are there multiple date formats accepted? I've been using 8-1-2023, for example (I'm on Windows, if that matters)

Reply:
Yes. It's any format that can be read by [`DateTime.Parse`](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.parse?view=net-7.0#system-datetime-parse(system-string)).

I'll add a note.